src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -236,8 +236,9 @@ def cleanRSE(self, rse):
             ctx = gfal2.creat_context()
             gfal2.set_verbose(gfal2.verbose_level.names[self.msConfig['gfalLogLevel']])
         except Exception as ex:
-            msg = "RSE: %s, Failed to create gfal2 Context object. Skipping it in the current run."
-            self.logger.info(msg, rse['name'])
+            msg = "RSE: %s, Failed to create gfal2 Context object. " % rse['name']
+            msg += "Skipping it in the current run."
+            self.logger.info(msg)
             raise MSUnmergedPlineExit(msg)
```

Comment:
I believe this exception is caught in the `_execute` method, right? If so, shouldn't we simply remove the "info" log record in the line above and rely only on the `_execute` method?
Actually, to me it looks like a hard failure, since it prevents us from deleting files, no? In that case, an "error" or "exception" log message would make more sense. What do you think?

Reply:
any trigger of `MSUnmergedPlineExit` is caught and logged as a woraning anyway, So I'd just remove the log.info here.

Reply:
Todor, don't you want to know why it failed to create this context object (whatever it is)?

Reply:
As you wish, but we risk to have thousands of those dumps  in case of an external failure. I was thinking of moving this object up in the class instance - then it would be feasible to have it as an exception. But it would need additional testing. I see here and there warnings about problems with re-usability of the `gfal.context` objects. And that's why I kept its recreation here on every RSE cleaning.

Reply:
done